cmake/onnxruntime_unittests.cmake

```diff
@@ -530,7 +530,7 @@ if (onnxruntime_BUILD_HOSTING)
   )
   if(NOT WIN32)
     if(HAS_UNUSED_PARAMETER)
-      set_source_files_properties("${TEST_SRC_DIR}/hosting/util_tests.cc" PROPERTIES COMPILE_FLAGS -Wno-unused-parameter)
+      set_source_files_properties("${TEST_SRC_DIR}/hosting/json_handling_tests.cc" PROPERTIES COMPILE_FLAGS -Wno-unused-parameter)
```

Comment:
Why do we need this?

Reply:
I try to isolate protobuf related code to a separate file. Otherwise all files reference to `util.h` will need `-Wno-unused-parameter`. In current implementaion, only this file need this special switch.